### ISSUE TITLE
Redesign Concepts

### ISSUE THREAD

**Issue by JohnCoates** [JohnCoatesOSS/Limitless#28]
MataNataM has kicked off the redesign with some great designs posted on the chat. I'm opening this issue to encourage remixes, comments, critiques, other concepts, etc.

![limitless-light](https://cloud.githubusercontent.com/assets/967800/20697313/fcbed9fc-b5ac-11e6-9c83-e91ba2e9d608.png)
![limitless-dark](https://cloud.githubusercontent.com/assets/967800/20697314/fcd1d32c-b5ac-11e6-84bf-7dc5f5c89367.png)

SVG files are posted here: https://gitter.im/JohnCoatesOSS/Limitless?at=58354582cc0ea2cf778593a9

**Comment by Shugabuga** [comment]
I didn't do anything for a dark mode as it would follow the conventions of the renders above, but here is an idea (refined) for a modernized Home screen based off of the Packages concept (note: the Limitless icon is just a placeholder):

![home](https://cloud.githubusercontent.com/assets/10801851/20733548/f2c742c6-b661-11e6-9c54-542b60d9578b.png)


[[Download SKETCH file]](https://github.com/JohnCoatesOSS/Limitless/files/620540/home.zip)

**Comment by nathanaccidentally** [comment]
I like these. Especially the simpler more "Apple like" approach to the navigation bar.

**Comment by kirb** [comment]
My criticism:
• Icons are too big. Makes it seem like it's trying to make up for not having enough to fill the page. What else could be added?
• Nav bar buttons could be simplified to a settings cog and reload arrow.
• There should be a divider above the find extensions button (maybe? Not sure if it'll look worse)
• Needs a design for alerts. Can't find a screenshot on google for you, but a few times a red box has appeared under the welcome heading to show an urgent notice.

**Comment by Ankiiii** [comment]
I really like that design. I hope to see a feature to enable dark mode. Kinda like Twitter.

**Comment by Shugabuga** [comment]
@Kirb thanks for the criticisms! I'll keep them in consideration when I have time to work on it.

Definitely agree with using glyphs for the settings/reload though, I just need to find good ones. Hopefully IconJar has something.

**Comment by kirb** [comment]
Try Noun Project. iOS also has some built in (including the reload glyph, but it might be too thick).

**Comment by MataNataM** [comment]
![image](https://cloud.githubusercontent.com/assets/19202363/20863034/bfdf3118-b9c5-11e6-98a3-4d14c93d06f4.png)
![image](https://cloud.githubusercontent.com/assets/19202363/20863033/bfdf0c6a-b9c5-11e6-8d5e-7412dd7dddd8.png)

**Comment by Shugabuga** [comment]
Love the login screen, but I am still uneasy about the banner. Maybe a gray infinity sign and text with a normal background? And maybe ditch the divider between the big buttons.

Besides the banner, it's looking strong! 😃

**Comment by Shugabuga** [comment]
Besides the banner, it's looking strong! 😃

**Comment by MataNataM** [comment]
Hah thanks. I'll try to work my way around that and come up with something less harsh.
Maybe only the normal icon with text next to it?

**Comment by MataNataM** [comment]
Time to experiment again 😃

**Comment by Shugabuga** [comment]
> Maybe only the normal icon with text next to it?

This sounds good!

**Comment by leo98gomez** [comment]
Another or that could be redesigned is the installation screen. Instead of just being plain black it could be Translucent like the tweak TranslucentCydia does

**Comment by Shugabuga** [comment]
@leo98gomez I agree on this front. Maybe even hiding this screen or make it minimizable.

**Comment by Shugabuga** [comment]
I made another concept for the main menu of Limitless and a mockup on how minimized installs could work:

[[Download SKETCH file]](https://github.com/JohnCoatesOSS/Limitless/files/629950/home.zip)


![home](https://cloud.githubusercontent.com/assets/10801851/20870167/3fed9846-ba51-11e6-8586-fe8fcc49bd8b.png)
![installling tweak](https://cloud.githubusercontent.com/assets/10801851/20870166/3fed5494-ba51-11e6-87e7-4202890421cf.png)

**Comment by Shugabuga** [comment]
Another version that is less "toyish" requested by @JohnCoates:

![home_installing](https://cloud.githubusercontent.com/assets/10801851/20871221/4682e08e-ba61-11e6-8218-f87c0efc61be.png)
![home](https://cloud.githubusercontent.com/assets/10801851/20871222/46964cbe-ba61-11e6-98f8-691c278c0369.png)

[[Download SKETCH file]](https://github.com/JohnCoatesOSS/Limitless/files/629981/SketchFile.zip)

**Comment by Shugabuga** [comment]
[[Download SKETCH file]](https://github.com/JohnCoatesOSS/Limitless/files/629981/SketchFile.zip)

**Comment by maximehip** [comment]
Who can do a concept for settings view ?

**Comment by MataNataM** [comment]
I guess I can do it. What options do we need put into the settings pane?

**Comment by maximehip** [comment]
- Landscape mode 
- Night Mode
- Refresh at launch 
- Shake to respring

I don't have more ideas

**Comment by MataNataM** [comment]
![image](https://cloud.githubusercontent.com/assets/19202363/20927190/0b0ec95a-bbc9-11e6-8f1a-d5234c9f8881.png)
Here's what i got. Let ne know what you think. If you have any critique make sure to shoot it my way :)

**Comment by MataNataM** [comment]
![limitless settings 02](https://cloud.githubusercontent.com/assets/19202363/20928911/951e3958-bbd0-11e6-866b-0c161d1d4f2f.png)
Here's what I got. Let me know what you think. If you have any critique make sure to shoot it my way :)

**Comment by maximehip** [comment]
Good work !

**Comment by CloneClasher** [comment]
Color scheme wise, are we for sure, using purple? I see that some people have used it and I don't really like it.

EDIT: Also, what is needed? As in, settings, main page, etc.

**Comment by Shugabuga** [comment]
The color isn't set in stone. Purple was just nothing more than an idea. Likely will just use blue for now.

If you want to do something with design, [here is a list of things you could do](https://github.com/johncoatesoss/limitless/issues?q=is%3Aopen+is%3Aissue+label%3Adesign).
If you want to dive into coding for Limitless, [here's that list](https://github.com/JohnCoatesOSS/Limitless/issues?utf8=✓&q=is%3Aissue%20label%3A%22coder%20wanted%22%20).

**Comment by MataNataM** [comment]
Hey, here's a recent edit I've made, somewhat inspired by iOS 10.
[[SVG file available here]](https://www.dropbox.com/s/dc4moe35hq530p0/Limitless%20home%2010.svg)
![img_2016](https://cloud.githubusercontent.com/assets/19202363/23970278/f6c731f6-09d1-11e7-9f0a-e3ec88dd7278.PNG)

**Comment by Shugabuga** [comment]
The "Featured" box looks weird IMO. Also if we want this to look like iOS 10, there is usually a bold header (see Music app).

**Comment by MataNataM** [comment]
@Shugabuga yeah I kinda see your point, I was just playing around and thought I'd share :) 
About the bold text - I don't know who at apple decided it was good idea to use bold headers, it's so ugly 🙈

**Comment by Shugabuga** [comment]
Idk. The same person who removed the headphone jack? /s

**Comment by aesign** [comment]
![Limitless](https://user-images.githubusercontent.com/35091482/36859561-0b105e0a-1d7f-11e8-93f3-2b787f9de9b6.png)
My UI Proposal (Tab Bar icons are temporary)

**Comment by JohnCoates** [comment]
@aesign I really like this design. I do feel the colors are maybe too bright and look too playful. I'd say consider making the icons flatter and using more muted tones

**Comment by aesign** [comment]
![limitless](https://user-images.githubusercontent.com/35091482/36864448-04c14b00-1d8d-11e8-9a1f-935373d4360d.png)

[Sketch File](https://github.com/JohnCoatesOSS/Limitless/files/1772470/Limitless.zip)

**Comment by aesign** [comment]
![Manage Account](https://user-images.githubusercontent.com/35091482/36919280-12bec5ea-1e5d-11e8-809c-4a16f4ee9743.png)

@JohnCoates

**Comment by DaJakerBoss** [comment]
Okay @JohnCoates for the time being can we at least colour the nav bar? It would make _some_ distinction between Cydia and Limitless.
